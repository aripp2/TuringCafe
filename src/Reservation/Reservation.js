import React from 'react';
import './Reservation.scss';

const Reservation = ({ id, name, date, time, number, cancelReservation }) => {

  return (
      <div className='card'>
        <h3>{name}</h3>
        <h4>{date}</h4>
        <h4>{time}</h4>
        <h4>Number of Guests: {number}</h4>
        <button
          onClick={() => cancelReservation(id)}
        >Cancel</button>
      </div>
    )
}

export default Reservation;