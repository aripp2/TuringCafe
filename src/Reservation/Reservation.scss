.card {
  height: 300px;
  width: 200px;
  background-color: white;
  margin-top: 25px;
  
}