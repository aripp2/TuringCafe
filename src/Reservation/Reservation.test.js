import React from 'react';
import { shallow } from 'enzyme';
import Reservation from './Reservation';

describe('Reservation', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Reservation
      id={2}
      name='Amy'
      date='10/10'
      time='2:00'
      number={2}
      cancelReservation={jest.fn()}
      />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
