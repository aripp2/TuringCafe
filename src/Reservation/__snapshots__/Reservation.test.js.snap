// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reservation should match the snapshot 1`] = `
<div
  className="card"
>
  <h3>
    Amy
  </h3>
  <h4>
    10/10
  </h4>
  <h4>
    2:00
  </h4>
  <h4>
    Number of Guests: 
    2
  </h4>
  <button
    onClick={[Function]}
  >
    Cancel
  </button>
</div>
`;
